src/pkg/util/user/identity_test.go

```diff
@@ -0,0 +1,31 @@
+package user
+
+import "testing"
+
+func TestGetPwUID(t *testing.T) {
+	_, err := GetPwUID(0)
+	if err != nil {
+		t.Errorf("Failed to retrieve information for UID 0")
```

Comment:
How about making this `t.Fatalf`, and then adding a test that `Name == "root"`, `UID == 0`, `GID == 0`? And similar for `TestGetPwNam()`.